crates/key-server/src/errors.rs

```diff
@@ -14,6 +14,7 @@ pub enum InternalError {
     InvalidSignature,
     InvalidSessionSignature,
     InvalidCertificate,
+    InvalidSDKVersion,
```

Comment:
```suggestion
    DeprecatedSDKVersion,
```
?

Reply:
I've added both. Invalid is given in case the SDK version is not given or is not parseable. Deprecated is given in case the version is not in the required range.